Examples/UIExplorer/UIExplorer/AppDelegate.m

```diff
@@ -36,7 +36,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/Examples/UIExplorer/UIExplorerApp.includeRequire.runModule.bundle?dev=true"];
+  jsCodeLocation = [NSURL URLWithString:@"http://bimawa.local:8081/Examples/UIExplorer/UIExplorerApp.includeRequire.runModule.bundle?dev=true"];
```

Comment:
please don't change this - it looks like you have several commits stacked in this PR - can you move the unrelated ones to new PR's and squash the remaining ones into one commit for this PR?